scripts/generate-config.js

```diff
@@ -0,0 +1,113 @@
+const fs = require('fs');
```

Comment:
suggestion: 
you can use `typescript` for scripts too:
just run `ts-node file.ts` instead of `node file.js`

Reply:
I don't think it's necessary in such a small script. TS will introduce a lot of boilerplate.